RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Push Jobs Client Release Notes
 
+## 3.0
+
+Versions of Chef Push Jobs 3.0 and later may only be used under the terms of the [Chef EULA](https://www.chef.io/end-user-license-agreement/) or another commercial agreement with Chef Software.
+
+Dependency updates:
+- Ruby 2.6.6
```

Comment:
The CVEs fixed here should get noted. It's all in the chef infra client release notes if you need the links

Reply:
it's going to be a really long list based on how far these have bumped.

Reply:
@tas50 do you know of a good source for the lists? https://www.cvedetails.com/vendor/12872/Libarchive.html seems wrong.

Reply:
yeah the list of CVEs between Ruby 2.4.5 and 2.6.6 are too confusing. I'm just gonna say CVEs fixed.